source/system/math.ooc

```diff
@@ -33,6 +35,8 @@ extend Long {
 	}
 	maximum: func (other: This) -> This { this > other ? this : other }
 	minimum: func (other: This) -> This { this < other ? this : other }
+	maximumValue ::= static INT64_MAX
+	minimumValue ::= static INT64_MIN
```

Comment:
Hmm generally yeah, except we (meaning I) seem to have always put them at the top in this file. Let's be consistent.